NEWS.md

```diff
@@ -14,8 +14,10 @@
 * Updated examples to use `scda.2022`.
 * Added R session information into a link in the footer of `teal` applications.
 * Added data hashing step using `rlang` instead of `digest` package to calculate the hash (which has been moved from `teal.data` and `teal.slice`). There is now an explicit hashing check in the reproducible code output.
+* Removed deprecated function `root_modules`.
```

Comment:
The NEWS entry has been added here but was not removed from line 41, please remove it.

Reply:
and deprecated functions can be grouped in a single entry.